cypress-tests/cypress/support/pageBuilder/pbCreateCategory.ts

```diff
@@ -0,0 +1,47 @@
+import { gqlClient } from "../utils";
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
+            pbCreateCategory(data: {
+                name: string;
+                slug: string;
+                layout: string;
+                url: string;
+                // Add other fields as needed
+            }): void;
+        }
+    }
+}
+
+const MUTATION = /* GraphQL */ `
+    mutation pbCreateCategory($data: PbCategoryInput!) {
+        pageBuilder {
+            category: createCategory(data: $data) {
+                data {
+                    slug
+                    name
+                    layout
+                    url
+                }
+            }
+        }
+    }
+`;
+
+Cypress.Commands.add("pbCreateCategory", data => {
+    return cy.login().then(user => {
+        return gqlClient
+            .request({
+                query: MUTATION,
+                variables: { data },
+                authToken: user.idToken.jwtToken
+            })
+            .then(response => {
+                if (response.pageBuilder.category.error) {
+                    console.error(response.pageBuilder.category.error);
+                }
```

Comment:
`console.error(response.pageBuilder.category.error);` won't work because `error` was not requested in the mutation. We're only requesting `data`. Let's add `error` then or simply remove this altogether.

I'm thinking we do the latter and later introduce a smarter way of dealign with errors (if we'll need this at all).

Reply:
We also want to actually return `response.pageBuilder.category.data`.